community/project committers.md

```diff
@@ -35,3 +35,11 @@
   | Xianming Lei    | DiDi Global  |
   | Chenyu Zheng    | Ke Holdings  |
   | Qing Liu        | Lalamove     |
+  | Enrico Minack   | G-Research   |
+  | Felix Cheung    | NVIDIA       |
+  | Junping Du      | Datastrato   |
+  | Liu Xun         | Datastrato   |
+  | Ricky Ma        | Tencent      |
+  | Weiwei Yang     | Apple        |
+  | Yang Jie        | Baidu        |
```

Comment:
Yang Jie is duplicated, too.

Reply:
Yang Jie is to be removed? Jie Yang is correct?

Reply:
Yes, Jie Yang is correct.

Reply:
done